pkg/api/vcs.go

```diff
@@ -0,0 +1,29 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/grafana/grafana/pkg/services/vcs"
+)
+
+func (hs *HTTPServer) storeObjInVCS(ctx context.Context, kind vcs.Kind, uid string, obj interface{}) error {
+	if hs.VCS == nil || hs.VCS.IsDisabled() {
+		return nil
+	}
+
+	dashJson, err := json.Marshal(obj)
```

Comment:
```suggestion
	dashJson, err := json.MarshalIndent(obj, "", "  ")
```
To have a better file stored in the VCS (and have a beautiful demo 😇).